src/css/expanded-image.css

```diff
@@ -4,6 +4,7 @@
   border-radius: 5px;
   cursor: pointer;
   transition: 0.3s;
+  padding: 20px 20px 20px 20px;
```

Comment:
This wouldn't work because it's an inline element.
You should probably also add
`display: inline-block` or `display: block`

And also I would add only top and bottom padding. (`padding: 20px 0`), unless you want to achieve something else

Reply:
Maybe my suggestion is not ideal or goes too wide...I floated an image in the What's New and this has the wrapping issue: 
https://deploy-preview-1393--hardcore-allen-f5257d.netlify.app/hazelcast/6.0-snapshot/ask-ai

Reply:
Though it does seem to fix the particular issue which is what I was looking for...

Reply:
Oh, now I see :)
Then it does work indeed. Previously I was checking a non-floating image and it had no effect on it.